gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubImpl.java

```diff
@@ -16,9 +16,9 @@
 
 package com.google.cloud.pubsub;
 
+import static com.google.api.client.util.Preconditions.checkArgument;
```

Comment:
Wrong import. You can configure IntelliJ to avoid import suggestions from `com.google.api.client.util.*`